spec/views/sign_up/recovery_codes/show.html.slim_spec.rb

```diff
@@ -43,4 +21,30 @@
 
     expect(rendered).to have_css('.step-3.active')
   end
+
+  context 'when render has no dependencies' do
+    before do
+      render
+    end
+
+    it 'has a localized heading' do
+      expect(rendered).to have_content t('headings.recovery_code')
+    end
+
+    it 'informs the user of importance of keeping the recovery code in a safe place' do
+      expect(rendered).to have_content t('instructions.recovery_code')
+    end
+
+    it 'contains link to continue authenticating' do
+      expect(rendered).
+        to have_xpath("//input[@value='#{t('forms.buttons.continue')}']")
```

Comment:
would it be possible to use `have_link` in these expectations instead? I usually try to use xpath as a last resort since it tends to need to be changed a lot as the page's markup changes